### DISCUSSION TITLE
Where to download old releases and binaries?

### DISCUSSION THREAD

**Discussion by GregVernon** [snl-dakota/dakota#38] [Q&A] [answered]
The old place for downloading binaries was https://dakota.sandia.gov/downloads/, however that page now says to access them from https://github.com/snl-dakota/dakota/releases.  However this page only has a list of 5 "releases" that are all pre-releases issued in the last 5 days, and the "tags" are unsearchable, leading to quite a few button clicks on `Next` to find the [6.17](https://github.com/snl-dakota/dakota/releases/tag/6.17.0) or older tags.  These tags, including the [6.18](https://github.com/snl-dakota/dakota/releases/tag/v6.18.0) tag, only contain source code -- no binaries are available.

**Comment by jadamstephens** [comment]
Greg,

As you noticed, we are transitioning to offering downloads on GitHub. Recent releases WERE there, but due to a bug in the job we use to prune old stable releases, they were accidentally deleted. I'll get them back up ASAP. Thanks for letting us know about the problem.

Adam

**Reply by GregVernon** [reply]
As a follow-up, in the meantime I've been trying to build from the Dakota GitHub repository, but there don't appear to be "build from repo source" instructions.  The current [`INSTALL`](https://github.com/snl-dakota/dakota/blob/devel/INSTALL) instructions appear to be based on the assumption that one has downloaded one of the `*.src.tar.gz` distributions.  Are there instructions for building if I've done `git clone`?  In particular, I'm not sure how to get TPLs

**Reply by briadam** [reply]
Hi Greg,

This part of the manual is a work in progress but aims to address that: https://snl-dakota.github.io/docs/6.18.0/users/compiling/getsource.html

Brian

**Comment by jadamstephens** [comment, marked as answer]
Greg,

The [6.18.0 release](https://github.com/snl-dakota/dakota/releases/tag/v6.18.0) is back up. I'll add a few older versions as time permits today. Are you looking for a particular version?

Adam

**Reply by GregVernon** [reply]
I was looking for 6.17, but only because I wasn't quite sure if 6.18 was _actually_ released, since I didn't see any news on the main Dakota website.  Currently the main page makes it look like 6.17 is the latest:

![image](https://github.com/snl-dakota/dakota/assets/868479/c0312674-3bec-4952-87cc-192dd5a44ed8)

**Reply by jadamstephens** [reply]
We haven't officially announced the release of 6.18 for Sandia red tape reasons. We hopefully will be able to do so very soon. Regardless, the 6.18 binaries on the release page are fine. I also just (re)added the 6.17 packages

Adam
